### ISSUE TITLE
Assess PUI site accessibility and identify issues for remediation

### ISSUE THREAD

**Issue by erussey** [emory-libraries/aspace#103]
Please assess the accessibility of the public user interface site (archives-test.libraries.emory.edu) and identify issues where accessibility standards are not being met.  Part of the task of this ticket is to identify what standards to apply.

**Comment by maxdmayhew** [comment]
from ticket #102 https://accessibility.princeton.edu/how/content/headings

**Comment by maxdmayhew** [comment]
- [ ] Correct heading nested structure
- [ ] Color contrast
- [ ] Header links
- [ ] images have descriptions
Every "type" of page will need to be reviewed, will be using an accessibility checker as well.

**Comment by CB987** [comment]
The Chrome Lighthouse assessment tool found: 
```
[id] attributes on active, focusable elements are not unique
input#multiselect-item```
This is in the resources results, which may not be in our control, but coming from Lyrasis.

```Background and foreground colors do not have a sufficient contrast ratio.
div.pui-search-text```
/archives.libraries.emory.edu

```Links do not have a discernible name
a.logo-badge```
the footer logo on archives.libraries

```Heading elements are not in a sequentially-descending order```
goes from h1 to h3 without an h2 

on a PUI search results page, archives.libraries/search?
```ARIA IDs are not unique
select#limit.form-control.fill-column
input#limit```

```<dl>'s do not contain only properly-ordered <dt> and <dd> groups, <script>, <template> or <div> elements.```

on a resource page,

**Comment by CB987** [comment]
I'm starting with Google Chrome Lighthouse.
[Here are also the WCAG guidelines.](https://www.w3.org/WAI/WCAG21/quickref/)

**Comment by CB987** [comment]
Axe dev tools reports:
on home page:
9 "serious" errors. 4 text with insufficient color contrast and 5 links that need more info for screen readers. 
and 9 "moderate" errors. lack of heading hierarchy, and landmarks issues, including [lack of one main landmark](https://dequeuniversity.com/rules/axe/4.6/landmark-one-main?application=AxeChrome),

**Comment by erussey** [comment]
@CB987 : When the analysis is done, please let me know so I can set up a meeting w/ you @maxdmayhew @rotated8 to think through which of these things makes sense to fix and which should be submitted to the ASpace community as an addition to the core code.  (It might be that we can contribute that code to them as part of the ticket.  Not sure how that would work.)

**Comment by CB987** [comment]
Sure. Some of the things that we're finding are not coming from our overrides/plugin, so it makes sense to submit those to Lyrasis. I would think that most of the things within our own plugins should be pretty easy to maintain. But yes, let's discuss and prioritize. I think I am getting a pretty good picture of the across-the-board issues.

**Comment by CB987** [comment]
Testing home page, /repositiories, /repositories/resources, /search, /repositories/#/resources, repositories/#/resources/#

Manually checked:
Page titles: look good. 
Images have alt text: logo in both public and staff headers yes, logo in footer being not technically and <img> does not but did get called out for being a link without a name.
Headings: not appropriately nested on all pages.
- Front page doesn't have an h2
- help page, /repositories, /repositories/resources don't have an h1
Color contrast:
- home page links in paragraph on gray background do not have enough contrast (Element has insufficient color contrast of 3.97 (foreground color: #336be6, background color: #e7eaf1, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1) 
- also pretty much all the links listed below as well.
Links: should not be only identified by color contrast. 
- home page paragraph, 
- /help, 
- /repositories/7/resources/1042 under subjects,
Tabbing through links, looks good.

**Comment by CB987** [comment]
@erussey I think this is done. Do you want to schedule the appt with everyone or do you want me to?

**Comment by erussey** [comment]
@tmill29 is scheduling a meeting to discuss the above.

**Comment by tmill29** [comment]
The meeting has been scheduled for Tuesday, May 9th at 2 pm

**Comment by erussey** [comment]
Marking this as done as we had the meeting above and decided on which issues to proceed with.  Resulted in #143, #142, #145, #141, #144
